server/src/test/java/org/cloudfoundry/identity/uaa/oauth/UaaTokenStoreTests.java

```diff
@@ -206,27 +209,27 @@ void retrieveNonExistentToken() {
     }
 
     @Test
-    @SuppressWarnings("java:S2925")
-    void cleanUpExpiredTokensBasedOnExpiresField() throws InterruptedException {
+    void cleanUpExpiredTokensBasedOnExpiresField() {
         int count = 10;
-        store = new UaaTokenStore(dataSource, Duration.ofSeconds(3));
+        store = new UaaTokenStore(dataSource, givenMockedExpiration());
         String lastCode = null;
         for (int i = 0; i < count; i++) {
             lastCode = store.createAuthorizationCode(clientAuthentication);
         }
         assertThat(jdbcTemplate.queryForObject("SELECT count(*) FROM oauth_code", Integer.class), is(count));
 
+        jdbcTemplate.update("UPDATE oauth_code SET expiresat = ?", System.currentTimeMillis() - 60000);
```

Comment:
And you have to put back this statement that you were happy to have deleted.

Reply:
yes because your argument yestday , but here I dont see the cleaner fullfilled